src/main/java/com/alinab/taskOneGOFpatterns/behavioral/chainOfResponsibility/Numbers.java

```diff
@@ -0,0 +1,26 @@
+package com.alinab.taskOneGOFpatterns.behavioral.chainOfResponsibility;
+
+public class Numbers {
+
+    private final int numberOne;
+    private final int numberTwo;
+    private final String calculationRequest;
```

Comment:
also we can use @fielddefaults for decrease repeatable code

Reply:
Done